pkg/arm/arm_test.go

```diff
@@ -0,0 +1,75 @@
+package arm
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	acsapi "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/config"
+	"github.com/openshift/openshift-azure/pkg/util/fixtures"
+	"github.com/sirupsen/logrus"
```

Comment:
nit: group imports as discussed elsewhere

Reply:
@kargakis I feel like if you really want this enforced, a lint tool is in order.  I don't think it's worth the effort.